docs/shipping/Code/http.md

```diff
@@ -138,5 +138,173 @@ Allow some time for data ingestion, then open [Open Search Dashboards](https://a
+  -u '<https://otlp-listener.logz.io/v1/logs>' \\
+  -H 'Authorization: Bearer <Logzio-Token-Logs>' \\
+  -H 'user-agent: logzio-protobuf-logs' \\
+  -d @export_logs_request.json
+```
+
+Breakdown:
+
+* `I`: Points to the location of the OpenTelemetry Protobuf definitions.
+* `i`: Specifies the Protobuf request type (`ExportLogsServiceRequest`).
+* `o`: Specifies the Protobuf response type (`ExportLogsServiceResponse`).
+* `u`: URL of the Logz.io OTLP listener endpoint. Adjust the URL for your region using Logz.io [region settings](https://docs.logz.io/docs/user-guide/admin/hosting-regions/account-region/).
+* `H`: Include headers like the Authorization token and user-agent.
+* `d`: Specifies the JSON file containing the log data.
+
+## Prepare the JSON Data
+
+You need to create the `export_logs_request.json` file, which contains the structure of the log data to be sent to the OTLP listener. The required fields in the log request are as follows:
+
+
+```bash
+{
+  "resourceLogs": [
+    {
+      "resource": {
+        "attributes": [
+          {
+            "key": "service.name",
+            "value": { "stringValue": "example-service" }
+          }
+        ]
+      },
+      "scopeLogs": [
+        {
+          "scope": {
+            "name": "example-scope"
+          },
+          "logRecords": [
+            {
+              "timeUnixNano": "<timestamp>", // e.g., 1727270794000000000
+              "severityNumber": "SEVERITY_NUMBER_INFO",
+              "severityText": "INFO",
+              "body": { "stringValue": "Log message here" }
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}
+```
+
+Key fields:
+
+* `timeUnixNano`: A required field that represents the timestamp in nanoseconds since epoch (e.g., `1727270794000000000` for a future time). This needs to be manually set, but you can automate it in future feature requests.
+* `severityNumber`: Log severity level (e.g., `SEVERITY_NUMBER_INFO`).
+* `body`: The log message content.
+
+## Adjust the Region URL
+
+You must adjust the OTLP listener URL based on your Logz.io account region. Find the correct endpoint for your region [here](https://docs.logz.io/docs/user-guide/admin/hosting-regions/account-region/).
+
+For example, if you're in the US region, your endpoint might be:
+
+```bash
+-u '<https://otlp-listener.logz.io/v1/logs>'
+```
+
+For EU region:
+
+```bash
+-u '<https://otlp-eu.logz.io/v1/logs>'
+```
+
+## Full Command Example
+
+```bash
+protocurl -v \\
+  -I ~/Downloads/proto/opentelemetry-proto \\
+  -i opentelemetry.proto.collector.logs.v1.ExportLogsServiceRequest \\
+  -o opentelemetry.proto.collector.logs.v1.ExportLogsServiceResponse \\
+  -u '<https://otlp-listener.logz.io/v1/logs>' \\
+  -H 'Authorization: Bearer <Your-Logzio-Token>' \\
+  -H 'user-agent: logzio-protobuf-logs' \\
+  -d @export_logs_request.json
+```
+
+## Sample Output in Console
+
+When you run the command, you should see an output like this in your console:
+
+```bash
```

Comment:
I noticed this part is repetitive, so we can remove it since it’s already mentioned earlier in this guide.